README.md

```diff
@@ -1,38 +1,167 @@
+    { id: 1, offsetY: 104, data: null },
+    // ...
+    { id: 6, offsetY: 495, data: null },
+  ];
+}
+```
+
+## Components details
+
+### SidenoteWrapper
+
+| Argument          | Type   | Default | Mode           | Description                                                                                                        |
+| ----------------- | ------ | ------- | -------------- | ------------------------------------------------------------------------------------------------------------------ |
+| activeSidenoteId  | Number | null    | Controlled     | Selected sidenote id (presence of this property will enable controlled mode)                                       |
+| defaultSidenoteId | Number | null    | Non-controlled | Default selected sidenote id on wrapper load                                                                       |
+| gutter            | Number | 0       | Both           | Vertical spacing added between every sidenotes                                                                     |
+| items             | Array  | -       | Both           | List of items corresponding to wanted sidenotes                                                                    |
+| onSelect          | Action | -       | Both           | Action raised when using `select` exposed action on `Sidenote` component                                           |
+| onSidenotesMoved  | Action | -       | Both           | Action raised after sidenotes finished moving (called after load, selection, unselection, or sidenote size change) |
+| onUnselect        | Action | -       | Both           | Action raised when using `unselect` exposed action on `Sidenote` component                                         |
+
+This component expose two variables.
+
+- [0] is a `Sidenote` component instance
+- [1] is the current item to process (as `SidenoteWrapper` iterate over `@items`)
+
+### Sidenote
+
+| Argument | Type   | Default | Description                                                                                         |
+| -------- | ------ | ------- | --------------------------------------------------------------------------------------------------- |
+| id       | Number | -       | Sidenote id used as the unique identifier of this sidenote (should be unique)                       |
+| offsetY  | Number | -       | Preferred vertical position of the sidenote (the sidenote will position at this offset if possible) |
+| data     | Any    | -       | Custom data (this will be passed to `@onSelect` and `@onUnselect` of the wrapper)                   |
+
+This component expose one variable which has 3 properties
+
+- `isActive` is a boolean value that reflects if current sidenote is the one selected
+- `select` is an action to use to select current sidenote (will behave differently depending on the mode used)
+- `unselect` is an action to use to unselect current sidenote (will behave differently depending on the mode used)
```

Comment:
```suggestion
- `onSelect` is an action to use to select current sidenote (will behave differently depending on the mode used)
- `onUnselect` is an action to use to unselect current sidenote (will behave differently depending on the mode used)
```

Reply:
No, Sidenote component expose 2 actions select & unselect to trigger SidenotesWrapper onSelect & onUnselect listeners